flintrock/ec2.py

```diff
@@ -3,6 +3,8 @@
 import sys
 import time
 import urllib.request
+import base64
```

Comment:
Yes.  As you said I didn't use `base64` module in this commit, but I need it in next commit. 

`ec2.create_instances()` supports automatically base64 encoded `UserData`, however, 
`ec2.meta.client.request_spot_instances()` doesn't seem to support that.

 I think I should give base64 encoded string `user_data` to `ec2.meta.client.request_spot_instances()` `UserData` parameter. 

This is what I'm dealing with in above 'error when requesting the spot instances' problem.